src/Workspaces/CSharp/Portable/CodeGeneration/CSharpSyntaxGenerator.cs

```diff
@@ -1202,21 +1202,31 @@ public override IReadOnlyList<SyntaxNode> GetNamespaceImports(SyntaxNode declara
 
         public override SyntaxNode InsertNamespaceImports(SyntaxNode declaration, int index, IEnumerable<SyntaxNode> imports)
```

Comment:
@mattwar Thanks for explaining. I've added a commit. Is this what you'd like to see?